.coveragerc

```diff
@@ -3,7 +3,7 @@ branch = True
 data_file = .coverage
 source=edx_exams
 omit =
-    edx_exams/settings*
+    edx_exams/settings/*
     edx_exams/conf*
```

Comment:
does this also need an additional back slash?

Reply:
Good catch. Thanks! I don't think it affected the coverage because there is no `conf` directory, so I didn't notice it. I'll remove this line.

Reply:
Oh, wait. It's probably just matching the `conf.py` file. I think it's working as intended :).

Reply:
Ohh got it. Thank you!